### PR TITLE
feat: added testing and improved sidebar

### PR DESCRIPTION
**Pull request by Enhmunh-E**
100% coverage for sidebar component
now sidebar shows the selected sidebar in a different color.
Screen Shot 2022-08-09 at 14 43 21
Screen Shot 2022-08-09 at 14 43 35

### CODE REVIEW

File: frontend/__tests__/src/components/Sidebar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import SideBar, { menuItems } from "../../../src/components/Sidebar";
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```
Comment (Narantsatsralt05) [inline frontend/__tests__/src/components/Sidebar.test.tsx:3]: end ygad ignore hiisen be

File: frontend/src/pages/index.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { Container } from "@mui/material";
 const HomePage: NextPage = () => {
   return (
     <Container>
-      <div>home page</div>
+      <div>Home</div>
```
Comment (Narantsatsralt05) [inline frontend/src/pages/index.tsx:7]: eniig yah gej oorchilson be?

**Review by Narantsatsralt05 — COMMENTED** [review]
test iinhee file iinhaa deer 

```js
/*
1. ... case iig coverloson
2. ... case iig coverloson
*/
```

geed bichmeer baina ingeh ym bol yg ymar case uudiig cover hiisen gedgiig amarhan unshij chadah geed baina.

**Comment by Narantsatsralt05** [comment]
Nice PR thanks for descriptive message and image 😃

File: frontend/__tests__/src/components/Sidebar.test.tsx (hunk 1 above)
Comment (Enhmunh-E) [reply frontend/__tests__/src/components/Sidebar.test.tsx:3]: ignore hiin jest.spyOn functiong ashiglasan bolno.

**Review by Narantsatsralt05 — APPROVED** [review]

**Pull request merged**